manifests/scenarios/gmp-otel/1_collector.yaml

```diff
@@ -190,34 +187,86 @@ data:
       prometheus/bench:
         config:
           scrape_configs:
-          - job_name: otel-prom-bench
+          - job_name: PodMonitoring/gmp/avalanche/metrics
```

Comment:
Tried to do exactly match our relabelling, but looks like I broke it.. to be investigated. Any troubleshooting guidance for Otel, without Prometheus like target/service discovery UI?